pennylane/devices/qubit/simulate.py

```diff
@@ -279,9 +279,33 @@ def simulate(
 
     has_mcm = any(isinstance(op, MidMeasureMP) for op in circuit.operations)
     if circuit.shots and has_mcm:
-        return simulate_one_shot_native_mcm(
-            circuit, debugger=debugger, rng=rng, prng_key=prng_key, interface=interface
+        results = []
+        aux_circ = qml.tape.QuantumScript(
+            circuit.operations,
+            circuit.measurements,
+            shots=[1],
+            trainable_params=circuit.trainable_params,
         )
+        keys = jax_random_split(prng_key, num=circuit.shots.total_shots)
+        if qml.math.get_deep_interface(circuit.data) == "jax":
+            # pylint: disable=import-outside-toplevel
+            import jax
+
+            def vv(k):
```

Comment:
vv? What's that stand for?

Reply:
Nothing really, it is just a local function definition for vmap. Would you prefer we call it `simulate_partial`?